styles/home.style.desktop.css

```diff
@@ -0,0 +1,411 @@
+@media (min-width: 768px) {
+  /* styling the top navigation */
+
+  body {
+    max-width: 100vw;
+    position: relative;
+    overflow-x: hidden;
+  }
+
+  .top-nav {
+    display: block;
+    width: 100%;
+  }
+
+  main .container {
+    width: 100%;
+  }
+
+  .top-nav .container {
+    background-color: var(--color-primary);
+    color: #fff;
+    font-family: "Lato", sans-serif;
+    min-width: 100vw;
```

Comment:
- [ ] **Good design so far 👏 👏** However, it would be much better by making your project fully responsive to fit all the screen sizes, currently it gets disorganized on some points, I have recorded a video for your reference, Please Fix this to make a flexible webpage 🙏 

https://www.loom.com/share/f2ae6ade4c8e4409b6fb647adbe762b7


You may kindy follow this [link](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design) for more info on Responsive web design 😎